logstash-core/src/main/java/org/logstash/config/ir/expression/unary/Truthy.java

```diff
@@ -0,0 +1,28 @@
+package org.logstash.config.ir.expression.unary;
+
+import org.logstash.config.ir.SourceComponent;
+import org.logstash.config.ir.InvalidIRException;
+import org.logstash.config.ir.SourceMetadata;
+import org.logstash.config.ir.expression.Expression;
+import org.logstash.config.ir.expression.UnaryBooleanExpression;
+
+/**
+ * Created by andrewvc on 9/21/16.
+ */
+public class Truthy extends UnaryBooleanExpression {
+    public Truthy(SourceMetadata meta, Expression expression) throws InvalidIRException {
+        super(meta, expression);
+    }
+
+    @Override
+    public String toRubyString() {
+        return "!!(" + this.getExpression() + ")";
```

Comment:
You're right about that, I just did that for completeness. 

As far as the idea of `truthy` being confusing for users, I agree, but I think we can hide it. We need it because LIR is supposed to be non-ruby specific and capable of targeting languages with no notion of `truthiness`, like java, where an explicit nil/true/false check is required.